packages/lib/src/vis/shared/VisCanvasProvider.tsx

```diff
@@ -56,10 +56,17 @@ function VisCanvasProvider(props: PropsWithChildren<Props>) {
   const abscissaScale = getCanvasScale(abscissaConfig, visSize.width);
   const ordinateScale = getCanvasScale(ordinateConfig, visSize.height);
 
-  const worldToData = (vec: Vector2 | Vector3) =>
-    new Vector2(abscissaScale.invert(vec.x), ordinateScale.invert(vec.y));
-  const dataToWorld = (vec: Vector2 | Vector3) =>
-    new Vector2(abscissaScale(vec.x), ordinateScale(vec.y));
+  const dataToWorld = useCallback(
```

Comment:
Wrapping in `useCallback` here, after all, since everything else in the provider is memoised... It doesn't cost much anyway.